content/articles/docker-push-for-publishing-images-to-docker-hub/index.md

```diff
@@ -0,0 +1,125 @@
+![hero image](engineering-education/docker-push-for-publishing-images-to-docker-hub/hero.jpg)
+
+### Introduction
+Dockerfiles use has been on the rise in creating Docker containers. The untouched bit on Dockerfiles that rarely get a mention is its trivial advantage in creating a Docker image to which users can push to an online repository in Docker Hub. This makes it easier in sharing Docker images across various public and private repositories and registries giving users more flexibility in creating an earlier version of Docker images. We'll go through how to create a Dockerfile, create an account in Docker Hub from where we'll create a repository to which the built Docker image will be pushed. We'll also cover how to push Docker images to a registry.
+
+This article will be beneficial for individuals already familiar with Docker or readers with fundamentals of Containers will be able to pick it with cogent.
+
+### Prerequisites
+To get started, we have to install Docker on our system. Check out this amazing [article](/engineering-education/getting-started-with-docker/) on Getting Started with Docker.
+
+If you're not using Ubuntu be sure to review this official [documentation](https://docs.docker.com/engine/install/) on how to install Docker on your operating system environment.
+
+> Note that we will be using `Ubuntu 20.04` in this tutorial.
+
+### Creating a Dockerfile
+Before we publish a Docker image, it will be appropriate to build one right. So let's get to understand what a Dockerfile is first. A Dockerfile is a content file comprising of specific commands used to generate a Docker image. Let's proceed and create one, in your terminal create a directory and move into the directory created with the command below:
```

Comment:
Here, did you mean ''build one first or build one, right?'' It's not clear. Also, put the correct punctuation where necessary. For example, the sentence "Let's proceed and create one. In your terminal, create...." should have a full stop and not a comma.